### ISSUE TITLE
Multiple authors with the same affiliation

### ISSUE THREAD

**Issue by MRudey** [tudace/tuda_latex_templates#412]
Hi,
How do you suggest to handle multiple authors with the same affiliation using `\thanks`? My current workaround is to to do this:
```tex
\documentclass{tudapub}
% Preamble

% Switch on numbering for \thanks
\makeatletter
\let\@fnsymbol\@arabic
\makeatother

\begin{document}
\title{Some Title}
\author{
    Author A\thanks{Affil1 \newline}, %newline for better readability of affilations
    Author A\thanks{Affil2 \newline},
    Author A\thanks{Affil3},
    Author A$^1$,
    Author A$^1$,
    Author A$^2$
}
\maketitle
\end{document}

```
This looks already quite good, however the font for the numbers in mathmode is not the same as for the `\thanks`.
![Screenshot 2022-12-09 145220](https://user-images.githubusercontent.com/11921755/206717567-b083ab0a-cb81-4e3d-8748-b8eabd9aa71a.png)
Is there an official option to do this? Something like `\inst` for `tudabeamer`. I also tried using `\footnotemark[]` which creates an error.

**Comment by TeXhackse** [comment]
Hmm. My first idea was to do this using `\label`/`\footref` which I just learned it's not possible. (At least not with adding the functionality. KOMA-Script itself does not.)

`\inst` in beamer is doing nothing more than a `\textsuperscript`  This will probably be annoying when using a different bumbering mechanism but arabic numbers. With those you would do a `\textsuperscript{1}` instead of the mathmode switching. 

But I think it's a good point to add a separate macro to support the symbols using the arabic number as a reference.

Beside this … The ` \let\@fnsymbol\@arabic` ist not a good idea … I just found that it's internally hardcoded, which is also not solved well. So I will add an internally command to be redefined with the next update instead. 

Additionally the \newline shouldn't be placed in the `\thanks` it destroys the markup. One would prefer to redefine the internal command:

```
\ExplSyntaxOn\makeatletter
\newcommand*{\ptxcd_title@footnote}[2][1]{
	\textsuperscript{\@fnsymbol{#1}}#2\\par
}
\makeatother\ExplSyntaxOff
```

I am planning to rename some of the internals to make them easier accessible in the preparation of a Design adjustment. I will post here in case I do so to keep your adjustments working.

**Comment by TeXhackse** [comment]
Hmm. My first idea was to do this using `\label`/`\footref` which I just learned it's not possible. (At least not with adding the functionality. KOMA-Script itself does not.)

`\inst` in beamer is doing nothing more than a `\textsuperscript`  This will probably be annoying when using a different bumbering mechanism but arabic numbers. With those you would do a `\textsuperscript{1}` instead of the mathmode switching. 

But I think it's a good point to add a separate macro to support the symbols using the arabic number as a reference.

Beside this … The ` \let\@fnsymbol\@arabic` ist not a good idea … I just found that it's internally hardcoded, which is also not solved well. So I will add an internally command to be redefined with the next update instead. 

Additionally the \newline shouldn't be placed in the `\thanks` it destroys the markup. One would prefer to redefine the internal command:

```
\ExplSyntaxOn\makeatletter
\newcommand*{\ptxcd_title@footnote}[2][1]{
	\textsuperscript{\@fnsymbol{#1}}#2\\par
}
\makeatother\ExplSyntaxOff
```

I am planning to rename some of the internals to make them easier accessible in the preparation of a Design adjustment. I will post here in case I do so to keep your adjustments working.

**Comment by TeXhackse** [comment]
Hmm using the same structure as for beamer would break some other things. I'd still prefer the label/ref mechanism and will discuss that. 

Anyway the fnsymbol thing now has an own macro. I will edit the code snippet in the comment above after I did the release.

**Comment by MRudey** [comment]
Thanks for your effort! My attempts are just hacking around in LaTeX without any knowledge of the internals.

Will test everything as soon as released.

**Comment by TeXhackse** [comment]
The example already works it's included in the latest release from 2 weeks ago.

Adding the ref/label mechanism for this is a bit more complicated and in fact I think TUDa-CI is the wrong place to include it. I will discuss this with the main maintainer of the underlying class.

**Comment by MRudey** [comment]
> The example already works it's included in the latest release from 2 weeks ago.

Oh, great! I could not get it to work on the ShareLaTeX instance so I assumed that it's coming in the next release. Maybe it's not updated yet on the server.

**Comment by TeXhackse** [comment]
Thanks for the info. I will have a look in the afternoon. Usually they were fast with the updates. Will let you know when I tested.

**Comment by TeXhackse** [comment]
You are right … I requested an update there. In case you want to update yourself you can always place a copy of the latest [tudapub.cls](http://mirrors.ctan.org/macros/latex/contrib/tuda-ci/tex/tudapub.cls) within the project directory.
